lambda-assets/ca-registrator/app.ts

```diff
@@ -117,21 +117,63 @@ export const handler = async (event: any = {}) : Promise <any> => {
         throw new InformationNotFoundError(error.message);
       });
 
+    await s3Client.send(
+      new PutObjectCommand({
+        Bucket: bucketName,
+        Key: path.join(bucketPrefix || '', certificateId!, 'ca.public_key.pem'),
```

Comment:
use || or ??

Reply:
Bucket prefix is defined previously. Remove this OR operator.